uploader.json

```diff
@@ -1480,6 +1480,11 @@
             "pattern": [],
             "symbolic_name": "httpd_conf_d"
```

Comment:
This is part of the spec that you are changing and also needs to be the same spec name, `httpd_conf`.  Would you mind to please change this spec too.